ada/extensions/src/libadalang-helpers.ads

```diff
@@ -48,15 +49,20 @@ package Libadalang.Helpers is
       Description : String;
       --  Description for the application. Will be used in the help string.
 
-      with procedure Process_Context_Before (Ctx : Analysis_Context) is null;
-      --  This procedure will be called right before going through units
+      with procedure Process_Context_Before
```

Comment:
Originally, I didn't put the `Context` in the `Process_Unit` callback because you can reach it from the `Unit`. But if we're starting to have more `Context`, I wonder if we shouldn't put it in a record and pass it to every callback, because now users won't have access to the project tree from the `Process_Unit` callback.

Note: Just a thought, not review blocking